src/playwright/playwright.config.ts

```diff
@@ -52,7 +52,6 @@ export default defineConfig({
           testMatch: [
             // Migrated
             'src/*.test.accessibility.ts',
-            'src/*.*.test.accessibility.ts',
```

Comment:
Bonus. Apparently `testMatch` globs are greedy. So the glob above is enough to cover both cases.